src/components/Questionnaire/QuestionnaireList.tsx

```diff
@@ -78,13 +78,16 @@ const RenderCard = ({
           {questionnaireList.map((questionnaire: QuestionnaireDetail) => (
             <Card
               key={questionnaire.id}
-              className="overflow-hidden bg-white rounded-lg cursor-pointer transition-shadow transform hover:shadow-lg"
+              className="overflow-hidden bg-white rounded-lg cursor-pointer"
               onClick={() =>
                 navigate(`/admin/questionnaire/${questionnaire.slug}/edit`)
               }
             >
               <CardContent className="p-6 relative flex flex-col">
-                <div className="absolute top-4 right-4">
+                <div className="flex flex-row gap-2 justify-between items-center mb-4 border-b pb-2">
```

Comment:
needed since we've upgraded to tailwindcss v4;

```suggestion
                <div className="flex flex-row gap-2 justify-between items-center mb-4 border-b border-gray-200 pb-2">
```